src/Parser/Core/Modules/DeathTracker.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import { formatMilliseconds } from 'common/format';
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+const debug = true;
+
+class DeathTracker extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  deathTimestamp = 0;
+  resurrectTimestamp = 0;
```

Comment:
This could occur twice, how about naming this `lastResurrectionTimestamp` to make it more transparent that it might not include all resses?